### DISCUSSION TITLE
Header menu not working

### DISCUSSION THREAD

**Discussion by bhajay** [alshedivat/al-folio#2864] [General]
I am not sure if I am doing something wrong. Some links on the header menu are not linked correctly. 

Site: https://harish-research-lab.github.io/

- Blog (https://harish-research-lab.github.io/blog) works when you click the "blog" button
- Repositories (https://harish-research-lab.github.io/repositories/) works when you give the link. But if you press the repositories button from another page, it does not go there.

**Comment by bhajay** [comment]
This is now sorted. There is a line in header.liquid that says `p['url`... This should instead be `p.url`. Then this fixes it.

**Reply by george-gca** [reply]
Sorry, this was my mistake. I just fixed this in https://github.com/alshedivat/al-folio/commit/7c22e4800ab5e17a1401c70ac08bcbc74cb5b55e.
